src/handlers/channels/STAGE_INSTANCE_CREATE.ts

```diff
@@ -0,0 +1,9 @@
+import { eventHandlers } from "../../bot.ts";
+import type { StageInstance } from "../../types/channels/stage_instance.ts";
+import type { DiscordGatewayPayload } from "../../types/gateway/gateway_payload.ts";
+
+export function handleStageInstanceCreate(data: DiscordGatewayPayload) {
+  const payload = data.d as StageInstance;
```

Comment:
This is redundant now just do this `eventHandlers.stageInstanceCreate?.(data.d as StageInstance);`